azure-servicebus/src/main/java/com/microsoft/azure/servicebus/Utils.java

```diff
@@ -41,4 +43,44 @@ static void assertNonNull(String argumentName, Object argument) {
         if (argument == null)
             throw new IllegalArgumentException("Argument '" + argumentName + "' is null.");
     }
+    
+    static MessageBody fromSequence(List<Object> sequence)
+    {
+    	List<List<Object>> sequenceData = new ArrayList<>();
```

Comment:
>List<List<Object>> sequenceData [](start = 5, length = 31)

why does this have to be list<list<>>? Why not pass the sequence (which is already a list) directly?